src/views/Community/Editor.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { sendCommunityData } from '../../store/actions/community'
+import {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`reactstrap` import should occur before import of `../../store/actions/community`](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)